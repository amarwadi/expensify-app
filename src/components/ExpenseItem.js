import React from 'react';
import { Link } from 'react-router-dom';

const ExpenseItem = ({ id, description, amount, createdAt }) => (
    <div>
        <h3>
           <Link to={`/edit/${id}`}>
                {description}
            </Link>
        </h3>
        <p>{amount} at {createdAt}</p>
        
            
    </div>
);

export default ExpenseItem;