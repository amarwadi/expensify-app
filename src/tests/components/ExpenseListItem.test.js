import React from 'react';
import { shallow } from 'enzyme';
import ExpenseItem from '../../components/ExpenseItem';
import expensesList from '../../fixtures/expenses';

test('expense Item should render as expected', ()=>{
    const wrapper = shallow(<ExpenseItem {...expensesList[0]} />);
    expect(wrapper).toMatchSnapshot();
});
