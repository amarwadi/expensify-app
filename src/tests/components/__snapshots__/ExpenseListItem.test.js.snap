// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expense Item should render as expected 1`] = `
<div>
  <h3>
    <Link
      replace={false}
      to="/edit/1"
    >
      Gum
    </Link>
  </h3>
  <p>
    195
     at 
    0
  </p>
</div>
`;
